UPGRADING.md

```diff
@@ -38,8 +38,24 @@ The 2.0.0 release requires Python 3.6+.
 
 > **WARNING**: Breaking change
 
-The library was moved into `google.cloud.bigquery` namespace. Existing imports
-need to be updated.
+
+### Version 2.1.0
+
+The library had its old namespace restored, since importing from
+`google.cloud.bigquery` clashed with the `google-cloud-bigquery` library when the
+latter was also installed.
+
+The import paths that were changed in version `2.0.0` should be reverted:
+
+```py
+from google.cloud import bigquery_datatransfer
+from google.cloud import bigquery_datatransfer_v1
+```
```

Comment:
This assumes we will release the namespace adjustment in `v2.1.0`.

Version `2.1.0` can be viewed as a non-trivial "fix" of the v2.0.0 release, but if we want to release it as `v3.0.0`, please comment below.